.dockerignore

```diff
@@ -1,2 +0,0 @@
-server/node_modules/
```

Comment:
@FelixMalfait are you sure about removing this folder? Have you tried to build the docker and checked the image size?

Reply:
I see that you moved it to server folder. I don't think it works:
"When you run a build command, the build client looks for a file named .dockerignore in the root directory of the context"

Reply:
You are right! I had not tested it yet

Reply:
Thanks!